README.md

```diff
@@ -95,17 +111,6 @@ NavigationUI.shared.lineColor = .yellow
 
 * `routeControllerDidCancelNavigation`: Fired when the user taps `Cancel`. You are responsible for dismissing the UI
 
-## Examples
-
-We provide examples in Swift and Objective-C. Run `carthage update --platform ios` from the root folder and open MapboxNavigation.xcodeproj to try it out.
-
-### Running the example app
-
-1. If running in the simulator, you can simulate the user location by selecting `Debug` -> `Location` -> `City Bicycle Ride`
```

Comment:
@frederoni added a "Simulate Navigation" button which removes the need for the simulator-based location simulation